### PR TITLE
design bugs fixed

### PR DESCRIPTION
**Pull request by h-attila**
'lepj be' nagybetuvel
input mezonek ne legyen bordere, legyen 1px inset shadowja felul
input bal paddingja 13px
gombon ne legyen box-shadow
legyen vertikalisan kozepen a tartalom
ne logjon ki a feher a hatter alol semmilyen keparanyon

### CODE REVIEW

File: web/src/app.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ konnektApp.controller('registrationController', ['$scope', '$http', function ($s
 
 konnektApp.controller('loginController', ['$scope', '$http', function ($scope, $http) {
 
-  $scope.header = 'Jelentkezz be';
+  $scope.header = 'LÉPJ BE';
```
Comment (hubudibu) [inline web/src/app.js]: leave text in sentence case, convert to UPPERCASE via CSS

**Review by hubudibu — CHANGES_REQUESTED** [review]
+ remove box-shadow from button

File: web/src/app.js (hunk 1 above)
Comment (h-attila) [reply web/src/app.js]: ok

**Comment by h-attila** [comment]
desigb bugs fixed #2 :)

**Comment by h-attila** [comment]
conflicts resolved

File: web/index.html [hunk 2]
```diff
@@ -2,20 +2,14 @@
...
-
   </head>
   <body>
-    <img class="logo-feher-hosszu" src="/dist/images/logo-feher-hosszu.png" alt="Konnekt Kontakt Manager">
-    <div class="rectangle" ng-view>
-      <!-- registration, login code comes here -->
-
-    </div>
+    <div class="konnekt-logo"></div>
+    <div class="spacer"></div>
```
Comment (hubudibu) [inline web/index.html]: if you want to have space below the konnekt-logo, add some margin-bottom to it instead

**Review by hubudibu — CHANGES_REQUESTED** [review]

**Review by hubudibu — APPROVED** [review]

**Pull request merged**